doc/Clusters_from_Scratch/en-US/Ch-Installation.txt

```diff
@@ -508,20 +486,4 @@ Guidelines for Enterprise IP Multicast Address Allocation] paper.]
 ======
 
 The final /etc/corosync.conf configuration on each node should look
-something like the sample in Appendix B, Sample Corosync Configuration.
-
-[NOTE]
-======
```

Comment:
Dunno, on the one hand this this is useful info, on the other the document is aimed at people building their first cluster who dont care about compatibility.

I guess you're right.